ui.h

```
@@ -285,6 +289,7 @@ _UI_EXTERN char *uiWindowTitle(uiWindow *w);
  * @param title Window title text.\n
  *              A valid, `NUL` terminated UTF-8 string.\n
  *              Data is owned by the caller.
+ * @note This method is merely a hint and may be ignored by the system.
```

Are there really any mainstream cases the title is ignored?
I know there must be window managers out there which don't show titles (eg WMs for kiosk use?).
But it seems to me that anyone affected would likely be purposefully running some setup which hides titles and not be surprised by such behaviour.
So this comment might be more confusing than helpful (to windows/mac developers, at least)?

---

> Are there really any mainstream cases the title is ignored?

Not sure how mainstream my setup is, but I have been running i3wm (which is very popular) with the option `default_border pixel` for years. No title bars. I assume other titling WMs have similar options and for space saving purposes this would be reasonably common. Maybe that is not the case.

Thanks for looking through this btw!

---

Haha! Forgot to mention that I'm also an I3 user, although I've never fiddled much with the default config and I've never set mine up to hide window titles.
That was kind of my point: anyone who has configured their WM to not display titles probably isn't going to be too surprised that `uiWindowSetTitle()` doesn't have any visible effect, and the "this might be ignored" comment will just serve to alarm windows/mac/mainstream-WM-setup users.
Though... thinking about it, even if the title is hidden, the WM still probably stores it somewhere, right? So it's technically not ignored? Not 100% sure about that...

---

As to avoid any confusion, the `@note` here should explicitly say on `On unix platforms`.